src/dragonball/src/vm/mod.rs

```diff
@@ -799,6 +804,30 @@ impl Vm {
         }
     }
 
+    /// Resize MicroVM vCPU number
+    #[cfg(feature = "hotplug")]
+    pub fn resize_vcpu(
+        &mut self,
+        config: VcpuResizeInfo,
+        sync_tx: Option<Sender<bool>>,
```

Comment:
I am curious about how this parameter(sync_tx) works? 
( since now we always get a `None`, and we just pass it to vcpu_manager.resize_vcpu(..))

Reply:
Right now we are using async way to do cpu resize, later we could add sync way to do cpu resize through this Sender